shared/constants/fs.tsx

```diff
@@ -194,11 +206,11 @@ export const makeUploads = I.Record<Types._Uploads>({
   writingToJournal: I.Set(),
 })
 
-export const makeTlfs = I.Record<Types._Tlfs>({
-  loaded: false,
-  private: I.Map(),
-  public: I.Map(),
-  team: I.Map(),
+export const makeTlfs = (partial: Partial<Types.Tlfs>): Types.Tlfs => ({
```

Comment:
just a note about these helpers. before they  were strictly necessary as we needed the record factories, but now they're plain objects so their value is diminished. if we need all these fields filled then thats fine but also if its just used in a couple of places those things can just make the objects themselves

Reply:
Yeah a lot of these have to be populated, so I figured I'd just keep them. This particular one is only used in the reducer for initial state, so I'll get rid of it.